### ISSUE TITLE
Completion not triggered on . in With after numeric literal

### ISSUE THREAD

**Issue by Pilchie** [dotnet/roslyn#1293]
```VB
Class Program
    Public Property P As Long

    Sub M()
        With Me
            .P = 122
            .$$
        End With
    End Sub
End Class
```

Completion should be triggered at $$, but isn't.

**Comment by rchande** [comment]
Can you validate, @Pilchie ?

**Comment by Pilchie** [comment]
Yes, fixed by 3d31ec15ad3385d1a0a5c77aea6ca2674c7073b8
